engine/src/main/java/org/terasology/rendering/assets/shader/GLSLShaderFormat.java

```diff
@@ -31,9 +31,7 @@
 import org.terasology.assets.module.annotations.RegisterAssetFileFormat;
 import org.terasology.naming.Name;
 
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.Reader;
+import java.io.*;
```

Comment:
Your IDE probably did this. You should probably revert this change since importing whole io here is really not effective